tests/homePage/homePageHeader2.spec.js

```diff
@@ -29,7 +29,7 @@ test('Search suggestion in header', async ({ page }) => {
 test('Logo click redirect home page', async ({ page }) => {
```

Comment:
mix with above, have just one single e2e

Reply:
this is from another file, and it's a different test. Are you sure you want me to merge the 2 deifferent tests?